index.html

```diff
@@ -0,0 +1,42 @@
+           <div id="capture">
+               <caption id="img-caption">Neque porro quisquam est qui dolorem ipsum quia dolor sit amet, consectetur, adipisci velit</caption>
+           </div>
+        </div>
+        </div>
+        <div id="tributes">
+            <h2 id="tribute-info">Here's a time line of DR. John Maccarthur's life:</h2>
+            <ul>
+                <li class="tribute"> <strong>1969 -</strong>Nulla tortor sem, luctus at leo a, sollicitudin feugiat mi. Curabitur sapien metus, <br>
+                     vulputate eget turpis euismod, volutpat vehicula erat. Morbi eu sem velit</li>
+                <li class="tribute"> <strong>1973 -</strong>llentesque venenatis massa eget dui porta, nec ultrices dolor placerat. Vestibulum nec <br> dapibus tellus. Sed diam libero, rhoncus non nunc quis, condimentum dapibus felis. Nulla facilisi.</li>
+                <li class="tribute"> <strong>1978 -</strong>Donec luctus dictum orci sed semper. Sed accumsan ultrices porta. Nunc fringilla efficitur <br> velit, pellentesque scelerisque dui luctus a. Suspendisse in imperdiet orci. Etiam ultrices</li>
+                <li class="tribute"> <strong>1985 -</strong>ongue magna, sed porta erat venenatis vitae. Vestibulum ac augue congue, ullamcorper lacus <br> sed, vehicula odio. Duis viverra quis nisi eu hendrerit. Curabitur eget finibus nunc</li>
+                <li class="tribute"> <strong>1993 -</strong>in venenatis velit. Vestibulum vel leo vitae eros sodales porta. Fusce non <br> nulla at magna interdum elementum. Aliquam quis molestie tortor.</li>
+                <li class="tribute"> <strong>1997 -</strong>Praesent ultrices aliquam odio, ut vestibulum lectus bibendum eget. Vivamus libero turpis, aliquet at commodo nec, euismod quis diam. Ut pulvinar nibh id blandit pharetra.</li>
+                <li class="tribute"> <strong>2007 -</strong>Aliquam tempor odio vitae mattis vestibulum. Vestibulum sed tincidunt orci, mattis hend</li>
+                <li class="tribute"> <strong>2016 -</strong>r in dapibus purus, sed bibendum leo. Suspendisse tristique nibh tincidunt urna tincidunt, <br> vel fermentum mauris malesuada. Sed eleifend vitae turpis eu sollicitudin. Phasellus arcu neque, placerat eu nibh quis, pharetra vehicula magna. Praesent nunc purus, consequat in </li>
```

Comment:
use CSS classes to avoid code repetition. for instance, you could use a CSS class to bolden the li elements.

Reply:
- Inasmuch as your comment is helpful, I still suggest you use a persuasive language to point out issues. For example, You could say "I think using CSS classes to avoid code repetition would be appropriate" instead of saying "use CSS classes to avoid code repetition".